### ISSUE TITLE
High RAM use from Leftover FFMPEG processes when using write_videofile

### ISSUE THREAD

**Issue by Grenadier928** [Zulko/moviepy#1508]
I'm having an issue where writing a video to a file begins consuming all of my RAM. For example, during most of my program's execution, less than 4 gigs of RAM are in use. When I write to a video file, that number begins to drastically climb. Within less than a minute, its around 12 gigs. It typically crashes python around this point since my system runs out of memory. Looking at the task manager, I can see dozens of FFMPEG and conhost processes running when write_videofile is called. Typically, I can manage to write a file that is less than 2 minutes long, since it finishes writing to a file before I run out of memory. The resulting file is usually only around 50 megabytes, so I have no idea how writing to the file is taking this much memory.

I tried to address this by running my code in a loop so that it outputs files in 30 second increments which I could manually stitch together later. My reasoning was that I could free up extra memory between each write to avoid running out of memory. This is where things get weird:
```
iteration = 0

while (iteration * 30) <= FileDuration:
    goodExportClip = realFinalMovie.subclip(iteration * 30, (iteration * 30) + 30)
    goodExportClip.write_videofile(outputFile + str(iteration) + ".mp4", preset="ultrafast")
    
    goodExportClip.close()
    del goodExportClip
    gc.collect()
    iteration +=1
    print("Now sleeping for 8 seconds...")
    time.sleep(8)
```

What happens here is that although I free up a small amount of memory by closing the subclips after writing them out, the old FFMPEG and conhost processes from the write_videofile STILL remain in memory. As a result, I can still only export around 2 minutes of video before I run out of RAM.

#### Expected Behavior
FFMPEG processes to use a "normal" amount of memory during a write, and the FFMPEG/conhost processes to terminate after the write is finished

#### Actual Behavior
FFMPEG continues to consume memory until the system runs out of memory or the write finishes; whichever happens first. Old FFMPEG processes from write_videofile do not terminate when the write is finished, even when the clips write_videofile was called on are manaully deleted in memory.

#### Steps to Reproduce the Problem
Write a video file over a minute using write_videofile on Windows. The FFMPEG and conhost processes from the write will stay in memory, even after the write is finished and the clip has manually been closed and removed from memory.

#### Specifications

  - Python Version: 3.7.9
  - Moviepy Version: 1.0.3
  - Platform Name: Windows 10 Pro
  - Platform Version: Build 18363

**Comment by tburrows13** [comment]
Can you reproduce on v2.0.0.dev2 or the current master?

**Comment by Grenadier928** [comment]
I installed v2.0.0.dev2 and updated method names accordingly. The issue with RAM consumption persists, however, the console output now looks like this:
![3Capture](https://user-images.githubusercontent.com/78811645/109331361-becd0980-782a-11eb-9e74-41a26577e9d6.PNG)

It seems like the process repeatedly restarts, though I don't know why (the test code is not in any kind of loop). However, despite getting the "proc not detected" error repeatedly, a movie is still exported, but video only exists for the first ~3/4, which correlates to how far the last process was able to get before I ran out of memory.
